api/api.proto

```diff
@@ -1827,6 +1827,20 @@ service OpenStoragePool {
   }
 }
 
+// OpenStorageDiags service provides methods to manage diagnostic bundles
+service OpenStorageDiags {
+
+  // Collect diags based on the provided request
```

Comment:
I think this needs more documentation on how to use. Also, may want to return a `SdkDiagsCollectResponse` which has a `SdkJobResponse` or `SdkJob` object.

Reply:
Add documentation and also added a dedicated SdkDiagsCollectResponse.